### DISCUSSION TITLE
Notebook Mod: A Spanish modification of Notebook theme

### DISCUSSION THREAD

**Discussion by offray** [NicolasPetton/Notebook#35] [Q&A]
Hi,

Thanks for the Notebook theme. Based on it I have created the [Notebook Mod][notebook-mod], which mainly includes support for Spanish UI, Markdown and other minor stuff, but it is not quite a translation of Notebook. I wonder if is enough to link the theme in the Readme to raise awareness of such variant or there is any other way to improve collaboration between English and Spanish users of this theme.

Thanks,

Offray

[notebook-mod]: https://mutabit.com/repos.fossil/tiddlywiki/uv/seeds/notebook/notebook-mod.html

**Comment by NicolasPetton** [comment]
Hi @offray!

It's really cool to see a translation of the theme into Spanish! :+1: 

I think the best would be to include your Spanish translations directly into Notebook, as an optional package, using the same approach as with https://projectify.wiki, which is now available in 5 languages (including Spanish). 

I'd be happy to push on GitHub a skeleton for a Spanish language plugin, so you could open a pull request with your translations.
What do you think?

**Reply by offray** [reply]
Hi @NicolasPetton,

The approach I'm following is not quite a literal translation of all content in Notebook them. Just a home tilddler pointing to the original Notebook theme, the modifications on the Notebook Theme (Markdown support, TiddlyWiki's Spanish UI), different installing instructions directed towards non technical users and some short extra tiddlers. I have seen Projectify translations and they seem more detailed and closer to the original interface. So I don't know if creating a skeleton could be overkill for what I'm doing. Given such differences, how should we proceed?

**Reply by NicolasPetton** [reply]
Well, I'm not sure. The way you describe it, it looks like your modifications are built on top of Notebook, and are not related to the theme itself.

**Reply by offray** [reply]
Yes, they are built on top. That's why I thought that just a link in the Readme pointing towards the Mod theme would suffice. I think is important to raise awareness of this theme and showcase how it is used in different contexts and by different communities, as it was not my first finding when trying to look for a [mobile friendlier TiddlyWiki theme](https://mutabit.com/repos.fossil/offray-blog/uv/bliki/index.html#Searching%20for%20a%20mobile%20friendlier%20TiddlyWiki%20interface). 

(At some point I would like to combine the best of Notebook theme and [Bricks](https://tid.li/tw5/apps/bricks.html) [Plugins](https://tid.li/tw5/plugins.html), but that's for another conversation).

**Reply by NicolasPetton** [reply]
@offray I'd welcome a PR that links to your mod in the Readme :)

**Reply by offray** [reply]
Done. Thanks.
